vello_shaders/build.rs

```diff
@@ -136,3 +139,36 @@ fn write_msl(buf: &mut String, info: &ShaderInfo) -> Result<(), std::fmt::Error>
     writeln!(buf, "            }},")?;
     Ok(())
 }
+
+struct BuildScriptLog;
```

Comment:
It is semi obvious after reading more code, but having a doc comment here would be nice.